input/spec.md

```diff
@@ -45,3 +45,67 @@ Keys are as follows:
 * Affected_manufacturers : list of string-refs
 * Fixed_versions : list of string-refs
 * references : object of reference objects
+
+#### Exploitation details
+
+These are obtained from various sources, and give details of the type of vulnerability:
+
+* Surface:
+  * `local` - this vulnerability can be exploited by a user with physical access to the device
+  * `remote` - this vulnerability does not need physical access to the device
+  * `app` - this vulnerability can be exploited by a malicious application
+  * `webpage` - this vulnerability can be exploited by malicious code on a webpage
+  * `usb` - this vulnerability requires use of USB debugging to exploit
+  * `filesystem` - this vulnerability can be exploited by placing crafted files into a specific place in the filesystem
+  * `system-call` - this vulnerability can be exploited through system calls
+  * `sms` - this vulnerability can be exploited by sending the victim a malicious SMS message
+  * `mms` - this vulnerability can be exploited by sending the victim a malicious multimedia message
+* Vector:
+  * `insufficient-standards-verification` - a system component does not properly check standards, which allows a non-compliant app or feature to exploit the vulnerability
+  * `insufficient-memory-protection` - an app is allowed to access memory which it should not have access to
+  * `improper-verification` - a malicious app is able to impersonate a privileged app because the system does not properly verify whether it (or a message sent by it) is genuine
+  * `memory-corruption` - attacks via buffer overflows and similar methods
+  * `daemon-abusing` - exploiting a vulnerability in a daemon to gain privileged access to the system
+  * `property-space` - use of the system property space to escape the sandboxed environment
+  * `vulnerable-driver` - a device which driver can be exploited
+  * `file-permissions` - improperly set file permissions allow malicious apps to access or modify files which they should not have access to
+  * `kernel-vulnerability` - a vulnerability in the Linux kernel which can be exploited
```

Comment:
How does this differ from `system-call`?

Reply:
This was intended as a general kernel-related vector but I agree it's probably in the wrong place so I'll have another look over the ones where I've used it and try and replace it with something more specific